packages/markdown-docx/src/OoxmlTransformer.js

```diff
@@ -151,6 +151,17 @@ class OoxmlTransformer {
                 $class: `${NS_PREFIX_CommonMarkModel}Softbreak`,
             };
         case 'w:r':
+            if(element.elements[0].name==='w:rPr'){
+                let emphFound = element.elements[0].elements.some(subElement=>{
+                    return subElement.name==='w:i' && subElement.attributes['w:val'] === 'true';
+                });
+                if(emphFound){
```

Comment:
`emphasisedTextFound`

Reply:
@algomaster99 
I used the default JS and Ts formatters above.
Also, for prettier, there was no config file I didn't use it. I will use the default config for prettier that match the codebase.

Reply:
Yes, use prettier. It should take care of everything. Just make sure it doesn't change very unrelated things in codebase. For example, removes all the `;`. This is why I am asking you to write a config file. You can check the docs on how to write one. It's very straightforward.

Reply:
@algomaster99 
I have formatted using prettier. Let me know if there are any more changes required.

Reply:
Sure. Go ahead. Do that in a separate PR though. And then we can merge the changes with this PR.

Reply:
@algomaster99 I didn't made any drastic prettier change. Only small area of code needed proper spacing so I don't think it would need a PR.